include/quicly.h

```diff
@@ -351,6 +351,10 @@ struct st_quicly_stream_t {
      * stream id
      */
     quicly_stream_id_t stream_id;
+    /**
+     * host cid
+     */
+    quicly_cid_t *host_cid;
```

Comment:
I do not think that we need to do this, considering the fact that you can obtain CID from `quicly_stream_t::conn`.

Besides, I'd assume that we'd be removing (or refactoring) the code to access CID, because CID is something that changes during the connection.

Reply:
quicly_stream_t::conn can't be used,you will get "incomplete type".
quicly_conn_t is defined in source file not in header file

Reply:
Yes. That's intentional. Applications are expected to call `quicly_get_host_cid(stream->conn)`.

Reply:
sorry, I didn't notice this method.

Reply:
I found a comment ` /* TODO let the app set host cid after successful return from quicly_accept / quicly_connect */` and i misunderstood it...